README.md

```diff
@@ -1,3 +1,11 @@
-# JavaScript Complete YouTube
+Myntra Clone Repository
 
-![JavaScript Thumbnail](https://github.com/KG-Coding-with-Prashant-Sir/JavaScript_Complete_YouTube/assets/102736197/b969122e-5c69-4a81-8cb8-8252c2a47446)
```

Comment:
This repo does not have just the Myntra project. This is for the entire JS course. You can add the details for the project if you want. But we need to keep the title and the thumbnail.